baseapp/msg_service_router.go

```diff
@@ -125,6 +130,14 @@ func (msr *MsgServiceRouter) RegisterService(sd *grpc.ServiceDesc, handler inter
 			if err := req.ValidateBasic(); err != nil {
 				return nil, err
 			}
+
+			if msr.circuitBreaker != nil {
+				msgURL := sdk.MsgTypeURL(req)
+				if allowed := msr.circuitBreaker.IsAllowed(ctx, msgURL); !allowed {
```

Comment:
small simplification

```suggestion
				if !msr.circuitBreaker.IsAllowed(ctx, msgURL) {
```